packages/cli/src/commands/clean.ts

```diff
@@ -0,0 +1,102 @@
+import {readPackage} from 'read-pkg'
+import util from 'util'
+import {exec} from 'child_process'
+import enquirer from 'enquirer'
+import chalk from 'chalk'
+import ora from 'ora'
+
+export async function clean({id, winner}) {
+  const {prompt} = enquirer
+  const {log} = console
+
+  let experimentId: string = id
+  let experimentWinner: string = winner
+
+  if (!id) {
+    const response: {
+      id: string
+    } = await prompt({
+      type: 'input',
+      name: 'id',
+      message: 'Please enter the experiment ID you would like to clean up:',
+      validate: (i) => {
+        if (!i) {
+          log(chalk.red('Please enter a valid experiment ID'))
+          return false
+        }
+        return true
+      }
+    })
+    experimentId = response.id
+  }
+
+  if (!winner) {
+    const response: {
+      winner: 'a' | 'b'
+    } = await prompt({
+      type: 'select',
+      name: 'winner',
+      message: 'Please enter the winning side of the experiment:',
+      choices: ['A', 'B']
```

Comment:
Technically opticks supports a, b, c, d, etc https://github.com/viodotcom/opticks/blob/master/src/core/toggle.ts#L11

Should we allow it here too?

Reply:
I think for ease in the beginning we keep it as is and expand if we think its necessary (I think there has only been 1 MVT since I joined?) - we can always use the full command if needed:

```
opticks-cli clean --id=ab-test-123 --winner=c
```

Reply:
OK fair enough, though I think we should design it to be flexible, optimizing for the most used case is fine here.